src/tests/GC/API/GC/GetGCMemoryInfo.cs

```diff
@@ -72,9 +73,13 @@ static void MakeTemporarySOHAllocations()
         for (int i = 0; i < (totalTempAllocBytes / byteArraySize); i++)
         {
             byte[] byteArray = new byte[byteArraySize];
+            Consume(byteArray);
```

Comment:
```suggestion
            GC.KeepAlive(new byte[byteArraySize]);
```
Does this work? It should be less invasive that adding non-inlineable methods, etc.

(Similar for the other tests.)

Reply:
Just checked, yes, it preserves the allocation

Reply:
I'll change once the current test run finishes